src/OrbitGl/App.cpp

```diff
@@ -1074,64 +1075,81 @@ orbit_base::Future<void> OrbitApp::LoadModules(absl::Span<const ModuleData* cons
   };
 
   for (const auto& module : modules) {
-    futures.emplace_back(LoadModule(module).Then(main_thread_executor_, std::move(handle_error)));
+    futures.emplace_back(
+        LoadModuleAndSymbols(module).Then(main_thread_executor_, std::move(handle_error)));
   }
 
   return orbit_base::JoinFutures(futures);
 }
 
-orbit_base::Future<ErrorMessageOr<void>> OrbitApp::LoadModule(const ModuleData* module) {
+orbit_base::Future<ErrorMessageOr<void>> OrbitApp::LoadModuleAndSymbols(const ModuleData* module) {
+  return LoadModuleAndSymbols(module->file_path(), module->build_id());
+}
+
+orbit_base::Future<ErrorMessageOr<std::filesystem::path>> OrbitApp::LoadModule(
```

Comment:
Shouldn't this be moved next to the other overload, that this method calls?

Reply:
done.